common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import PureComponent from 'react-pure-render/component';
+
+export default class ClassicModal extends PureComponent {
+  static propTypes = {
```

Comment:
I'm not sure what you mean? Would it be 
```js
class ClassicModal extends PureComponent {
...
render() {
...
   }
}

ClassicModal.propTypes = {
...
}
```

I was following the format of `BugModal` for this, as I had never created a class from scratch with propTypes.